app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,7 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
+        android:usesCleartextTraffic="true"
```

Comment:
This may be needed cause sometimes imageURLs are not https and instead httpcleartraffic (technically security risk)